maths-club-pack/content/facilitator/coin-game.md

```diff
@@ -0,0 +1,56 @@
+# Coin Game 
+
+## Introduction
+
+This simple coin game serves as an introduction to probabilities in tennis. It is really worth playing the game first with the students before trying to answer the questions to ensure everyone understands the way the game works particularly when to stop playing.  
+
+Go through the answers to the first two questions before students try to fill in the table.  
+
+Watch students closely when they start filling in the table to ensure they are on the right track and help as necessary.  
+
+
+## Solution
+
+Which of the following scores are impossible in this game and why? 5-2, 7-4, 6-2,  
```

Comment:
It appears the author decided that some of the text should be in italics, could you please double check and fix, e.g.  
  _Which of the following scores are impossible in this game and why? 5-2, 7-4, 6-2,_
![image](https://user-images.githubusercontent.com/10515065/91466655-4d44dc00-e844-11ea-94d3-83580afb7d2f.png)